op-batcher/batcher/driver.go

```diff
@@ -254,12 +254,27 @@ func (l *BatchSubmitter) calculateL2BlockRangeToStore(ctx context.Context) (eth.
 func (l *BatchSubmitter) loop() {
 	defer l.wg.Done()
 
-	ticker := time.NewTicker(l.Config.PollInterval)
-	defer ticker.Stop()
-
 	receiptsCh := make(chan txmgr.TxReceipt[txData])
 	queue := txmgr.NewQueue[txData](l.killCtx, l.Txmgr, l.Config.MaxPendingTransactions)
 
+	// start the receipt/result processing loop
+	receiptLoopDone := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case r := <-receiptsCh:
+				l.Log.Info("handling receipt")
+				l.handleReceipt(r)
```

Comment:
would be good to add some context to the log line, e.g. the tx hash.

Reply:
The calls within (recordSuccess, recordFailed) do this more detailed logging; I added this mostly to be able to determine if tx failure arised from this call instead of another call location (recordFailed can be called from other places).   Tx hash isn't included in the failure case and didn't seem worth adding the conditional logic given it gets logged later anyway?

Reply:
I guess I could log the ID, eespecially when your multi-blob change goes in and it's type string?